org/postgresql/core/Version.java

```diff
@@ -0,0 +1,42 @@
+package org.postgresql.core;
+
+/**
+ * Enumeration for PostgreSQL versions.
+ */
+public enum Version {
+
+    v6_4(60400),
+    v6_5(60500),
+    v7_0(70000),
+    v7_1(70100),
+    v7_2(70200),
+    v7_3(70300),
+    v7_4(70400),
+    v8_0(80000),
+    v8_1(80100),
+    v8_2(80200),
+    v8_3(80300),
+    v8_4(80400),
+    v9_0(90000),
+    v9_1(90100),
+    v9_2(90200),
+    v9_3(90300),
+    v9_4(90400),
+
+    ;
+
+    private final int version;
+
+    private Version(int version) {
```

Comment:
Would you please use `String` constructor and `org.postgresql.core.Utils#parseServerVersionStr(String)` to convert values into `int`?

Or, alternatively, move all the version logic from `Utils` to this new `Version` enum. In other words, make `Utils#parseServerVersionStr` deprecated and keep it as a mere redirect to the implementation in `Version`.